lib/sql.js

```diff
@@ -48,8 +48,7 @@ var virtualRow = function (name) {
     }, function () {
         return SQLFunction.fromArgs(arguments);
     });
-     Object.setPrototypeOf(ident, null);
-     Object.setPrototypeOf(ret, ident);    
```

Comment:
looks like this was still the main change and the rest was just issues with the tests?